### DISCUSSION TITLE
prompt-engineering-for-developers 怎么被删掉了？

### DISCUSSION THREAD

**Discussion by yangmin4052** [datawhalechina/DOPMC#223] [问题求助] [answered]
我不能访问这个教程了 https://datawhalechina.github.io/prompt-engineering-for-developers/#/C3/5.%20%E5%9F%BA%E4%BA%8E%E6%96%87%E6%A1%A3%E7%9A%84%E9%97%AE%E7%AD%94%20Question%20and%20Answer

**Comment by siyuxin** [comment, marked as answer]
更新了 https://github.com/datawhalechina/llm-cookbook
